### ISSUE TITLE
The loss will be nan after long time training

### ISSUE THREAD

**Issue by ShihuaiXu** [hunglc007/tensorflow-yolov4-tflite#63]
Thanks for your great work, I have used your code to train my own data set.
But I found a issue that the loss will be nan after long time training of yolov4.
I will put the  loss of tensorboard to show it.
![2020-05-26 13-50-17屏幕截图](https://user-images.githubusercontent.com/35642740/82864759-e0295e00-9f57-11ea-9e33-35e100b6247b.png)

**Comment by robisen1** [comment]
did you find a solution? maybe the bounding box calculation?

**Comment by hunglc007** [comment]
I updated mish activation to fix nan loss issue.

**Comment by ghost** [comment]
> I updated mish activation to fix nan loss issue.

I'm still having the same issue. It starts off good and then after a long training time everything becomes `nan`.

<img width="421" alt="Capture d’écran 2021-01-07 180931" src="https://user-images.githubusercontent.com/44258725/103922060-8a36ae80-5113-11eb-92cb-eed48be56139.png">
